I wish_If only/Bohdanov_hw.md

```diff
@@ -0,0 +1,41 @@
+# Exercise 1. Please translate the sentences into English
+
+>Жаль что вы не пришли пораньше, чтобы оценить всю ущербность архитектуры предложенной студентом первокурсником.
+
+If only you had came up earlier to (evaluate / estimate) all the lameness of architecture that was proposed by first year student.
+
+>Обидно, что мы не должались его прихода, иначе бы мы были в курсе на каком этапе сейчас находится проект и что нам делать с этим дальше.
+
+If only we had waited for his arrival, otherwise we would be abreast on which stage is the project right now and what we should do with it futher.
+
+> Жаль что мы так и не встретились перед отъездом и не обсудили причину твоего увольнения, у меня много мыслей на этот счет и я хотел бы поделиться ими с тобой.
+
+I wish we had met before departure and discussed a reason of your quit, I had a lot of thoughts on this topic, and I wished to share it with you
```

Comment:
`the reason` - again we precise what kind of reason we are discussing
`resignation/quitting/firing/getting the sack` - as options but the form you used is wrong
`I have a lot of thoughts on this topic and want to share them with you` - it is present as he still has them and still wants